ast/compile.go

```diff
@@ -883,6 +883,60 @@ func compileSchema(goSchema interface{}) (*gojsonschema.Schema, error) {
 	return schemasCompiled, nil
 }
 
+//Checks if a JSON schema contains a specific type
+func containsType(schema *gojsonschema.SubSchema, s string) bool {
```

Comment:
This method doesn't look very useful if it is only used once and all it does is a String() comparison check

Reply:
yeah it was something more complicated before that did not work haha and now you're right it is very simple--i removed it